source/parser.py

```diff
@@ -211,8 +211,68 @@ def are_equal_list_authors(list1: List[Author], list2: List[Author]):
             return False
     return True
         
+
+# Neo4j database connection
+class Neo4jConnection:
+    def __init__(self, uri, user, password):
+        self._uri = uri
+        self._user = user
+        self._password = password
+        self._driver = None
+
+    def close(self):
+        if self._driver is not None:
+            self._driver.close()
+
+    def connect(self):
+        self._driver = GraphDatabase.driver(self._uri, auth=(self._user, self._password))
+
+    def query(self, query, parameters=None, db=None):
+        assert self._driver is not None, "Driver not initialized!"
+        session = self._driver.session(database=db) if db is not None else self._driver.session()
+        result = list(session.run(query, parameters))
+        session.close()
+        return result
+
+# Define a function to create nodes and relationships in Neo4j
+def create_neo4j_graph(grobid, cermine, neo4j_connection, url):
+    neo4j_connection.connect()
+    
+    # Create Author nodes
+    for author in grobid.authors:
+        create_author_query = "CREATE (:Author {name: $name, email: $email})"
+        neo4j_connection.query(create_author_query, {"name": author.name, "email": author.email})
+    
+    for author in cermine.authors:
+        create_author_query = "MERGE (:Author {name: $name, email: $email})"
+        neo4j_connection.query(create_author_query, {"name": author.name, "email": author.email})
+
```

Comment:
after the comparison, there should be only one input from author and paper, then the duplication issue can be solved